cmd/run.go

```diff
@@ -67,7 +67,7 @@ func SetupRunFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().Bool("force-checksum", false, "When true, the node will panic if the "+
 		"local state checksum differs from the network checksum reported by its peers.")
 	// Snapshot
-	cmd.PersistentFlags().Uint64("snapshot-block-height-period", 600000, "Set the snapshot epoch period. Snapshots are taken at block heights divisible by the period.")
+	cmd.PersistentFlags().Uint64("snapshot-block-height-period", lib.DefaultSnapshotEpochPeriodPoS, "Set the snapshot epoch period. Snapshots are taken at block heights divisible by the period.")
```

Comment:
Just defining as a constant to make it easier to re-use in other places. We'll eventually want to rip out this flag and enforce a consistent snapshot epoch length for PoS so that you can hypersync from any other hypersynced node.

Reply:
Yep that's what I figured when I read it the first time.